From fb89416918ceb4917e3a37f086486f7121a61e14 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Fri, 04 Jun 2021 08:11:30 +0300
Subject: [PATCH] [SW-0] preboot: increase preboot size by 1kb

Preboot size in greco has been increased by 1KB (65K -> 66K) to
accommodate static protection implementations.

Change-Id: I3daa4c6feb3e4557fa7c13cea1392eee8748175a
Signed-off-by: Karthikeyan K <karthikk@habana.ai>
---

diff --git a/arch/greco/common.h b/arch/greco/common.h
index 98a2999..25c6926 100644
--- a/arch/greco/common.h
+++ b/arch/greco/common.h
@@ -11,7 +11,7 @@
 #include <bitops.h>
 
 #define PPBOOT_SIZE			0x5000		/* 20KB */
-#define PREBOOT_SIZE			0x10400		/* 65KB */
+#define PREBOOT_SIZE			0x10800		/* 66KB */
 
 #define SRAM_SIZE			0x8000000	/* 128MB */
 #define HL_SP_SRAM_SIZE			0x10000
